### PR TITLE
Add MatMul testdata

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

**Comment by alalek** [comment]
Please rebase patch to resolve conflicts

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 1]
```diff
@@ -680,3 +680,63 @@ def forward(self, x):
...
+model = Expand(shape=[1, 2, 3, 4])
+save_data_and_model("expand_hw", x, model)
+
+class NormL2(nn.Module):
+    def __init__(self):
+        super(NormL2, self).__init__()
+
+    def forward(self, x):
+      norm = torch.norm(x, p=2, dim=1, keepdim=True)
+      clip = torch.clamp(norm, min=0)
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py]: Why we need to consider `Clip`?

Comment (l-bat) [reply testdata/dnn/onnx/generate_onnx_models.py]: Because customer model contains all these operations

Comment (dkurt) [reply testdata/dnn/onnx/generate_onnx_models.py]: Can you please show a subgraph? i.e. using [Netron](https://github.com/lutzroeder/netron)?

Comment (dkurt) [reply testdata/dnn/onnx/generate_onnx_models.py]: So I mean that for test that's fine but we cannot fuse it into a single Normalize layer - there should be Normalize + Clamp at least

Comment (l-bat) [reply testdata/dnn/onnx/generate_onnx_models.py]: Issue: opencv/opencv#16235
Screenshot from 2020-03-30 10-54-31
